stdlib/contrib/sranka/telegram/telegram.flux

```diff
@@ -39,13 +99,72 @@ message = (
-// `url` - string - URL of the telegram bot endpoint. Defaults to: "https://api.telegram.org/bot"
-// `token` - string - Required telegram bot token string, such as 123456789:AAxSFgij0ln9C7zUKnr4ScDi5QXTGF71S
-// `parseMode` - string - Parse mode of the message text per https://core.telegram.org/bots/api#formatting-options . Defaults to "MarkdownV2"
-// `disableWebPagePreview` - bool - Disables preview of web links in the sent messages when "true". Defaults to "false"
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with `channel`, `text`, and `silent`, as defined in the `message` function arguments.
+// endpoint sends a message to a Telegram channel using data from table rows.
+// 
+// ```
+// import "contrib/sranka/telegram"
+// 
+// telegram.endpoint(
+//   url: "https://api.telegram.org/bot",
+//   token: "S3crEtTel3gRamT0k3n",
+//   parseMode: "MarkdownV2",
+//   disableWebPagePreview: false,
+// )
+// ```
+// 
+// ## Usage
+// 
+// `telegram.endpoint` is a factory function that outputs another function.
+// The output function requires a `mapFn` parameter.
+// 
+// ### `mapFn`
+// A function that builds the object used to generate the POST request. Requires an `r` parameter.
+// 
+// `mapFn` accepts a table row (`r`) and returns an object that must include the following fields:
+// 
+// - `channel`
+// - `text`
+// - `silent`
+// 
+// For more information, see `telegram.message()` parameters.
+//
+// ## Parameters
+// 
+// - url: - string - URL of the Telegram bot endpoint. Default is `https://api.telegram.org/bot`.
+// - token: - string - (Required) Telegram bot token.
+// - parseMode: - string - [Parse mode](https://core.telegram.org/bots/api#formatting-options)
+//   of the message text.
+//   Default is `MarkdownV2`.
+// - disableWebPagePreview: - bool - Disable preview of web links in the sent message.
+//   Default is false.
+// 
+//   The returned factory function accepts a `mapFn` parameter.
+//   The `mapFn` must return an object with `channel`, `text`, and `silent`,
+//   as defined in the `message` function arguments.
+//
+// ## Examples
+// ### Send critical statuses to a Telegram channel
+// ```
+// import "influxdata/influxdb/secrets"
+// import "contrib/sranka/telegram"
+// 
+// token = secrets.get(key: "TELEGRAM_TOKEN")
+// endpoint = telegram.endpoint(token: token)
+// 
+// crit_statuses = from(bucket: "example-bucket")
+//   |> range(start: -1m)
+//   |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
+// 
+// crit_statuses
+//   |> endpoint(mapFn: (r) => ({
+//       channel: "-12345",
+//       text: "Disk usage is **${r.status}**.",
+//       silent: true
+//     })
+//   )()
+// ```
+// 
+// tag: notification-endpoints
 endpoint = (url=defaultURL, token, parseMode=defaultParseMode, disableWebPagePreview=defaultDisableWebPagePreview) => (mapFn) => (tables=<-) => tables
```

Comment:
I would expect the formatter to split this line.

Reply:
From what I've seen, the formatter likes arrow operations (`=>`) to be on a single line.